tests/drivers/gpio/gpio_basic_api/boards/frdm_mcxa156.overlay

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
We do not need this file. the testcase.yaml default supported this if you put the `arduino_gpio` in board support list